docs/source/getting-started/code/daml-after/User.daml

```diff
@@ -3,19 +3,16 @@
 
 module User where
 
--- MAIN_TEMPLATE_BEGIN
```

Comment:
I assume we only refer to this from the before state but not the after state. Please correct me if I'm wrong.

Reply:
That's right.